src/sentry/integrations/github/client.py

```diff
@@ -97,20 +97,20 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> List[str]:
             if msg == "Git Repository is empty.":
                 logger.warning(f"{repo_full_name} is empty.")
             elif msg == "Not Found":
-                logger.error(f"The Github App does not have access to {repo_full_name}.")
+                logger.warning(f"The Github App does not have access to {repo_full_name}.")
             else:
-                sentry_sdk.capture_exception(e)
+                logger.exception("An unknown error has ocurred.")
 
         return tree
 
     def get_trees_for_org(
-        self, org_slug: str, gh_org: str, cache_seconds: int = 3600 * 24
```

Comment:
I'm writing a script that is not tied to an org, thus, naming it `cache_key` is more generic.